### ISSUE TITLE
css:S4662 Tailwind directives and functions

### ISSUE THREAD

**Issue by unlight** [SonarSource/sonar-css#212]
https://tailwindcss.com/docs/functions-and-directives/
No errors on `@apply` tailwind directive, but for others:
Custom directives
```
@tailwind
@responsive
@screen
```
Custom functions:
```
theme();
```

Error css:S4662
`"at-rules" should be valid`

**Comment by vilchik-elena** [comment]
Hi,

You can configure the rule with tainwind-specific at-keywords, there is parameter for that.

What problem do you have with custom functions?

**Comment by clorichel** [comment]
May have missed it, but I'm not sure we can configure rules on SonarCloud. Just tried duplicating or extending the "Sonar way" rule, but then the only option is not to configure, but totally avoid `css:S4662`, which is definitely not what we'd want to do, isn't it?

I've "resolved as false positive" in the meantime: https://sonarcloud.io/project/issues?id=ganttlab_ganttlab&open=AXHPwCeB-_lUv_RZTL2l&resolutions=FALSE-POSITIVE&types=BUG

**Comment by vilchik-elena** [comment]
@clorichel You should copy "Sonar way" profile (not rule), and in the rule page you can "Change" the rule in the copied profile. If that makes you further trouble please create a thread in https://community.sonarsource.com/ (in this category https://community.sonarsource.com/c/help/sc/9) so that SonarCloud team could better assist you.
